pySDC/projects/DAE/sweepers/fully_implicit_DAE.py

```diff
@@ -28,6 +28,10 @@ def __init__(self, params):
         if 'QI' not in params:
             params['QI'] = 'IE'
 
+        quad_type = params['quad_type']
+        if quad_type == 'LOBATTO' or quad_type == 'RADAU-LEFT':
```

Comment:
You should rather check for `right_is_node`, see https://github.com/Parallel-in-Time/pySDC/blob/master/pySDC/core/Collocation.py#L62